src/server/graphql/httpGraphQLHandler.js

```diff
@@ -10,6 +10,8 @@ const {GQL_START, GQL_STOP} = ServerMessageTypes
 const {GQL_DATA, GQL_ERROR} = ClientMessageTypes
 export default (sharedDataLoader, rateLimiter, sseClients) => async (req, res) => {
   const {id: opId, type, payload} = req.body
+  const isTrebuchetClient =
```

Comment:
super lame that the CLI doesn't let us config what gets sent to the server.
if we're gonna ducktype it, can we ducktype it based on what it should look like?
eg `isCLI = req.body.query && req.body.operationName`

Reply:
You prefer I detect vanilla GraphQL over detecting the Trebuchet transport? I can do that... Let me take one more quick pass